openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -268,3 +268,13 @@ def test_registration_webinar_admin_delete_permission():
     Test that admins do not have permission to delete webinar registrations
     """
     assert not WebinarRegistrationAdmin.has_delete_permission('self', Mock())
+
+
+@pytest.mark.django_db
+def test_get_deleted_objects_webinar_admin(webinar_admin_instance, request, webinar, mocker):
+    """
+    Test get deleted objects for webinar admin is called to bypass permission
+    """
+    mocked_get_deleted_objects = mocker.patch.object(WebinarAdmin, 'get_deleted_objects')
+    webinar_admin_instance.delete_model(request, webinar)
+    assert mocked_get_deleted_objects.assert_called()
```

Comment:
The tests are failing because `delete_model` never calls `get_deleted_objects`. `get_deleted_objects` is called by ModelAdmin's `_delete_view`. Looking at Django source code will be helpful. We can discuss on call the ways to test this scenario.

Reply:
yes i resolved this you can review again